packages/twenty-front/src/modules/settings/hooks/useSettingsNavItem.ts

```diff
@@ -0,0 +1,18 @@
+import { useMatch, useResolvedPath } from 'react-router-dom';
+
+import { getSettingsPagePath } from '@/settings/utils/getSettingsPagePath';
+import { SettingsPath } from '@/types/SettingsPath';
+
+export const useSettingsNavItem = <Path extends SettingsPath>({
```

Comment:
I feel the useSettingsNavItem is not clear: can we make it computeSettingsNavItemConfig?
Also could you add an unit test on this one?

Reply:
I have replaced `useSettingsNavItem` with a `SettingsNavigationDrawerItem` component. It should be covered by the Settings pages stories. Do we want to add a specific story for the `SettingsNavigationDrawerItems` component as well?

Reply:
It does not seems to be necessary at this point. We might introduce a Settings coverage later to make sure that everything is covered. It seems that this is captured by other stories so we are good to go